mod/metric_answer_lookup/lib/answer/cardless_answers.rb

```diff
@@ -6,6 +6,10 @@ def self.included host_class
       host_class.extend ClassMethods
     end
 
+    def find_answer_card
+      Card[record_name, year.to_s, :value]&.left
```

Comment:
maybe add comment explaining why we get the value and then left (rather than going straight to the answer card)